pkg/monitoring/vmistats/collector_test.go

```diff
@@ -20,6 +20,8 @@
 package vmistats
```

Comment:
perfscale

Reply:
yup, made the change.

The package is called perfscale, and it contains these two new histograms.